nautobot/docs/configuration/optional-settings.md

```diff
@@ -234,6 +234,16 @@ Celery broker URL used to tell workers where queues are located.
 
 ---
 
+## CELERY_TASK_DEFAULT_QUEUE
+
+Environment Variable: `NAUTOBOT_CELERY_TASK_DEFAULT_QUEUE`
+
+Default: `'default'`
```

Comment:
Is this a change from prior releases? It was `celery` before, right? Is there anything we need to call out in the release-notes about the upgrade changing this?

Reply:
I added a section to the release notes overview. See comment above